base/special/rem_pio2.jl

```diff
@@ -0,0 +1,306 @@
+# This file is a part of Julia. Except for the rem_pio2_kernel, and
+# cody_waite_* methods (see below) license is MIT: https://julialang.org/license
+
+# rem_pio2_kernel and cody_waite_* methods are heavily based on FDLIBM code:
+# __ieee754_rem_pio2, that is made available under the following licence:
+
+## Copyright (C) 1993 by Sun Microsystems, Inc. All rights reserved.
+##
+## Developed at SunPro, a Sun Microsystems, Inc. business.
+## Permission to use, copy, modify, and distribute this
+## software is freely granted, provided that this notice
+## is preserved.
+
+const invpio2 = 6.36619772367581382433e-01
+pio2_1(::Type{Float64}) = 1.57079632673412561417e+00
```

Comment:
This one and the one below are like this because of how the method for `Float32` is going to look like. I could do it to all `const`s.